caddyconfig/httpcaddyfile/builtins.go

```diff
@@ -87,6 +87,7 @@ func parseTLS(h Helper) ([]ConfigValue, error) {
 	var folderLoader caddytls.FolderLoader
 	var certSelector caddytls.CustomCertSelectionPolicy
 	var acmeIssuer *caddytls.ACMEIssuer
+	var autoPolicy *caddytls.AutomationPolicy
```

Comment:
The only field being used is the KeyType; Instead, we can probably just make a `keyType` variable here.